parachain/runtime/src/lib.rs

```diff
@@ -1048,7 +1048,7 @@ impl_runtime_apis! {
             add_benchmark!(params, batches, relay, Relay);
             add_benchmark!(params, batches, vault_registry, VaultRegistry);
             add_benchmark!(params, batches, fee, Fee);
-            add_benchmark!(params, batches, nomination, Nomination);
+            // add_benchmark!(params, batches, nomination, Nomination);
```

Comment:
Probably should undo this?

Reply:
There are no nomination benchmarks at the moment, we should add in a separate PR.